### ISSUE TITLE
Fake Planck Likelihood

### ISSUE THREAD

**Issue by raziel2646** [baudren/montepython_public#116]
I'm trying to run a few chains with the fake_planck_likelihood but unfortunately, no matter what options I use, I don't get convergence. I've tried stepsizes between -f 1.3 to -f 2.4 and I've run the chains for N= ~100000 to ~500000 but my acceptance rate is always < 0.01. 

here's a sample of the commands I use to run MontePython:

```
mpirun -np 4 python montepython/MontePython.py run -f 1.5\
	-p input/lcdm_6.param  -o $output \
	-c covmat/fake_planck_lcdm.covmat -N 25000 --update 500 --silent
```

I would really appreciate some input on this.

**Comment by brinckmann** [comment]
Hi,
I need more information, can you share your param file?
Best,
Thejs

**Comment by raziel2646** [comment]
Hi,

Sorry for my late response. I'm just using the lcdm_6.param file as input. I'll copy the contents here:

```
#------Experiments to test (separated with commas)-----

# valid options are (so far): acbar, bicep, boomerang, cbi,
# hst, quad, sn, spt, fake_planck_bluebook,
# euclid_pk, euclid_lensing
# and clik_fake_planck (for Planck members)

data.experiments=['fake_planck_bluebook']
#data.experiments=['clik_fake_planck']
#data.experiments=['wmap','spt']

#------ Parameter list -------
# data.parameters[class name] = [mean, min, max, 1-sigma, scale, role]
# - if min max irrelevant, put to -1 or None (if you want a boundary of -1, use -1.0)
# - if fixed, put 1-sigma to 0
# - if scale irrelevant, put to 1, otherwise to the appropriate factor
# - role is either 'cosmo', 'nuisance' or 'derived'


# Cosmological parameters list
data.parameters['omega_b'] 	= [2.249,  1.8, 3, 0.016, 0.01,'cosmo']
data.parameters['omega_cdm'] 	= [0.1120, 0.1,   0.2, 0.0016,1,   'cosmo']
data.parameters['n_s']       	= [0.963,  0.9,1.1, 0.004, 1,   'cosmo']
data.parameters['A_s']       	= [2.42,   1.8,3, 0.038, 1e-9,'cosmo']
data.parameters['h']         	= [0.703,  0.6,0.8, 0.0065,1,   'cosmo']
data.parameters['tau_reio']    	= [0.085,  0.04,0.12, 0.0044,1,   'cosmo']

# Derived parameter list
#data.parameters['Omega_Lambda'] = [1,       0, 1, 1,1,  'derived']

# Other class parameters (fixed parameters, precision parameters, etc.)
#data.cosmo_arguments['sBBN file'] = data.path['cosmo']+'/bbn/sBBN.dat'


#------ Mcmc parameters ----
# Number of steps taken, by default (overwritten by the -N command)
data.N=10
# Number of accepted steps before writing to file the chain. Larger means less
# access to disc, but this is not so much time consuming.
data.write_step=5

```

my acceptance rate at the moment is around 0.002! 
Let me know if you need more information.

**Comment by brinckmann** [comment]
Hi,
I believe this is the same problem as issue #106, which is fixed in MontePython v3.0 on my github page.
Best,
Thejs

**Comment by raziel2646** [comment]
Thanks, I'll definitely check it out.

**Comment by raziel2646** [comment]
I just wanted to confirm that the issue is fixed in MontePython v3.0. 
Thanks again for your help.
